stable/object-bucket-claim/templates/object-bucket-claim.yaml

```diff
@@ -0,0 +1,25 @@
+{{- range $app, $config := .Values.dkp }}
+{{- if $config.enabled }}
+---
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: {{ $app }}-s3-bucket
+  {{- if $config.labels }}
+  labels: {{ toYaml $config.labels | nindent 4 }}
```

Comment:
since we are defining `object-bucket-claim.labels` in the template  file, should we use it here?

Reply:
added